rest-api/core/src/main/java/org/eclipse/kapua/app/api/core/model/CountResult.java

```diff
@@ -17,6 +17,9 @@
 import javax.xml.bind.annotation.XmlElement;
 import javax.xml.bind.annotation.XmlRootElement;
 
+/**
+ * @since 1.0.0
+ */
```

Comment:
maybe since 2.0.0?

Reply:
No, CountResult was existing before.
We added SetResult as a new class